internal/controllers/httpsedge_controller.go

```diff
@@ -315,6 +319,21 @@ func (r *HTTPSEdgeReconciler) setEdgeRouteResponseHeaders(ctx context.Context, e
 	return err
 }
 
+func (r *HTTPSEdgeReconciler) setEdgeTLSTermination(ctx context.Context, edgeID string, tlsTermination *ingressv1alpha1.EndpointTLSTerminationAtEdge) error {
+	client := r.NgrokClientset.EdgeModules().HTTPS().TLSTermination()
```

Comment:
Interestingly, it looks like you can also just set this on the edge itself instead of as an edge module. any idea if they are functionally equivalent? If so how you have it makes more sense than trying to keep it on the edge itself to me fwiw

Reply:
Yep. You can do that with all of the edge modules from what I can tell. However, from our API docs, there is no way to disable modules through the update EdgeRoute API. By updating the edge & edge route first and then handling the modules, we make it so we only have to have this logic in 1 place which seems cleaner to me and makes it so we only have to duplicate the edge logic for create/update

Reply:
ah nice, yeah that makes sense